var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -178,6 +178,8 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
         depends_on("umpire {0}".format(rocm_dep), when="+raja {0}".format(rocm_dep))
         depends_on("camp {0}".format(rocm_dep), when="+raja {0}".format(rocm_dep))
 
+    patch("exago-1.6.0.patch", when="@1.6.0")
```

Comment:
@balay we also have to patch some other exago versions for xSDK purposes, and so this is a really helpful example.

qq: should this patch be in 1.6.1? Seems like a logical change

Reply:
Yeah - best if all patches can go upstream in future bug/fix releases

Reply:
Made an exago issue to make sure this makes 1.6.1, thanks!